lib/payload.js

```diff
@@ -70,6 +70,10 @@ class Payload {
 			postmen.tmp_api_key = null;
 		}
 
+		if (config.csv) {
```

Comment:
no, it seems that the SDK doesn't provide such functionality to customize the headers.

I could do so, but doing so may involves more changes to this SDK.